news/0741cad6-3007-47f8-9c53-984e9116c7ff.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix: include sub-actions in tab completion
```

Comment:
```
Include sub-commands in tab completion.
```

Reply:
Done with de1ed6cbacc206f36bae911e5831625dee385dca, I also pushed another change cdf29afe8d468c0d94dd353a188c976a422c83a3. Let me know if things are looking ok and I'll squash my commits and `s/action/command/` there too :slightly_smiling_face: